Add osu! originals - osu!mania 7K World Cup 2023 news post (#8805)
ppy/osu-wiki: news/2023/2023-02-15-featured-artist-updates-mwc7k.md

---
layout: post
title: "osu! originals - osu!mania 7K World Cup 2023"
date: 2023-02-15 11:00:00 +0000
---

> As [MWC7K](/wiki/Tournaments/MWC/2023_7K) approaches its climax, explore **5** brand new songs created specifically for the tournament!

Reviewer: ```suggestion
As [MWC7K](/wiki/Tournaments/MWC/2023_7K) approaches its climax, explore **5** brand-new songs created specifically for the tournament!
```

![](/wiki/shared/news/banners/featured-artist.jpg)

But first, some context.

> Our Featured Artist program has expanded over the years, and we've only recently started dipping our feet into the depths of *original song commissions* (which we call [osu! originals](/wiki/osu%21_originals)). You might've seen a few osu! originals already via Featured Artist announcements or the recent [Twin Trials contest](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase).

Reviewer: ```suggestion
Our Featured Artist program has expanded over the years, and we've only recently started dipping our feet into the depths of *original song commissions* (which we call [osu! originals](/wiki/osu!_originals)). You might've seen a few osu! originals already via Featured Artist announcements or the recent [Twin Trials contest](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase).
```

> Our Featured Artist program has expanded over the years, and we've only recently started dipping our feet into the depths of *original song commissions* (which we call [osu! originals](/wiki/osu!_originals)). You might've seen a few osu! originals already via Featured Artist announcements or the recent [Twin Trials contest](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase).

Reviewer 2: hmm, i feel like this is dismissing the originals we've had created over the years a bit by saying "only recently". some of the first featured artists were actually brought on with originals as their initial contribution (IAHN for example). we've also had some amazing landino tracks, cysmix tracks, as well as other artists over the years. not sure how to change the wording in a way that would hint at this, but maybe would be nice to?

Reviewer: this is referring to original commissions for tournaments, but i guess the wording doesn't convey that properly

Reviewer: maybe something like this

```suggestion
Our Featured Artist program has expanded over the years, and we've only recently started dipping our feet into the depths of *original tournament track commissions* (also known under the [osu! originals](/wiki/osu!_originals) label). You might've seen a few osu! originals already via Featured Artist announcements over the years, or perhaps from the recent [Twin Trials contest](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase).
```

Reviewer 2: Good enough I think!

Thanks largely to [mangomizer](https://osu.ppy.sh/users/1893718) and each game mode's world cup staff, you may have noticed a bunch of osu! originals in our official tournaments too!

In case you missed any of the new songs, this article highlights every osu! original showcased in [osu!mania 7K World Cup 2023](/wiki/Tournaments/MWC/2023_7K)!

<div align="center">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/fa/mwc7k2023-exclusives.mp4" type="video/mp4" preload="none">
    </video>
</div>

### Toromaru - Uncharted Sky

> This high tempo jazzy tune by [**Toromaru**](https://osu.ppy.sh/beatmaps/artists/279) isn't what you'd expect from a rhythm game, but it's exactly what you need. This acted as a *Rice* pick in the [Semifinals map pool](/wiki/Tournaments/MWC/2023_7K#semifinals.1).

Reviewer: ```suggestion
This high-tempo jazzy tune by [**Toromaru**](https://osu.ppy.sh/beatmaps/artists/279) isn't what you'd expect from a rhythm game, but it's exactly what you need. This acted as a *Rice* pick in the [Semifinals map pool](/wiki/Tournaments/MWC/2023_7K#semifinals.1).
```

</audio>

### Hino Isuka vs. s-don - Glooorious!!

[**Hino Isuka**](https://osu.ppy.sh/beatmaps/artists/272) worked alongside **s-don** to design this \[glorious pun\] song used as a *Hybrid* pick in the [Finals map pool](/wiki/Tournaments/MWC/2023_7K#finals).

Try [its beatmap](https://osu.ppy.sh/beatmapsets/1935691) created (yet again) by [\_underjoy](https://osu.ppy.sh/users/2235750)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/272/Songs/Hino%20Isuka%20vs.%20s-don%20-%20Glooorious!!.mp3" type="audio/mpeg">
</audio>

### SAMString - Ataraxia

osu! player turned musician [**SAMString**](https://osu.ppy.sh/beatmaps/artists/273) has given back to osu! through this stellar electro tune used as a *Long Note* pick in the [Grand Finals map pool](/wiki/Tournaments/MWC/2023_7K#grand-finals).

Check out [the 9-star beatmap](https://osu.ppy.sh/beatmapsets/1939418) created by [Remuring](https://osu.ppy.sh/users/6522146) and [Jinjin](https://osu.ppy.sh/users/3360737)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/273/Songs/SAMString%20-%20Ataraxia.mp3" type="audio/mpeg">
</audio>

### katagiri - Kyuukou

> The [Grand finals tiebreaker](/wiki/Tournaments/MWC/2023_7K#grand-finals) is a coveted position for any beatmap, and this masterpiece by [**katagiri**](https://osu.ppy.sh/beatmaps/artists/176) and its has the slot secured.

Reviewer: ```suggestion
The [Grand Finals tiebreaker](/wiki/Tournaments/MWC/2023_7K#grand-finals) is a coveted position for any beatmap, and this masterpiece by [**katagiri**](https://osu.ppy.sh/beatmaps/artists/176) has the slot secured.
```
